skins/laika/tests/unit/store/payment_store.spec.ts

```diff
@@ -238,12 +238,12 @@ describe( 'Payment', () => {
 	} );
```

Comment:
Kinda out of place for this PR, but I assume this file got here when `npm run lint` ran.